Gopkg.toml

```diff
@@ -11,8 +11,8 @@ required = [
 ]
 
 [[constraint]]
-  name = "github.com/kubernetes-sigs/sig-storage-lib-external-provisioner"
-  version = "1.0.0"
+  name = "sigs.k8s.io/sig-storage-lib-external-provisioner"
+  branch = "master"
```

Comment:
do we want master or the 3.0.0-beta?

Reply:
I've removed this constraint section because it's a standalone package and we can review `Gopkg.lock` to see if it has been changed by accident.

If we want to use newer version we can update it to laster master by using `dep ensure -update` or a specify version by using `dep ensure -add sigs.k8s.io/sig-storage-lib-external-provisioner@<version>`.